cypress/e2e/02CreateNewAnalysis.cy.ts

```diff
@@ -0,0 +1,106 @@
+
+describe('Testing CreateAnalysis', () => {
+    it('Visits the Feeds Page', () => {
+      cy.visit('http://localhost:3000/feeds')
+        .wait(5000)
+  
+    })
+
+    it('01 Clicks on Create New Analysis', () => {
+        cy.get('.create-feed-button')
+          .click()
+    })
+
+    it('02 Types in the Feed Name', () => {
+        cy.get('input#feed-name').type('Testing Study')
+    
+      })
+
+    it('03 Types in the Feed Description', () => {
+        cy.get('textarea#feed-description').type('This is for testing purposes')
+          .wait(5000)
+          .screenshot()
+
+      })
+    
+    it('04 Goes to FeedType selection', () => {
+        cy.get('button.pf-c-button.pf-m-primary')
+        .eq(1)
+        .click()
+      })
+
+    it('05 Selects the feedtype', () => {
+        cy.get('label.pf-c-radio__label')
+        .eq(0)
```

Comment:
Somehow make sure this is actually the button you want, either via assertion or assign the element a test ID.